binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -241,7 +242,9 @@ public void initialize() {
             }
             onUpdate();
         } else {
-            logger.warn("Cannot connect to hue bridge. IP address or user name not set.");
+            String warn = "Cannot connect to hue bridge. IP address or user name not set.";
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.CONFIGURATION_ERROR, warn);
+            logger.warn(warn);
```

Comment:
Hm, if I configure my logger to log all messages of the Hue binding to a separate file, I would inspect this file first if my Hue is not recognized correctly.
IMHO it is ok, that a binding logs such stuff and send the information to other layer (without knowing the specific implementation of that layer).